cmake/project/CompilerSettings.cmake

```diff
@@ -27,4 +27,4 @@ elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel")
     # Intel C++
 elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
     # Visual Studio C++
-endif()
+endif()
```

Comment:
Unneeded change. Please add new line at the end of file and configure your IDE to always add a new line at the end of file. In VSCode it can be done by settings `"editor.formatOnSave": true` (or similar property).